src/main/java/frc/robot/commands/MoveTimed.java

```diff
@@ -0,0 +1,50 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class MoveTimed extends CommandBase {
+  private Drivetrain drivetrain;
```

Comment:
Within the new Command-based infrastructure, the RobotContainer class owns the Subsystems and all requests to "do" something to the robot should go through the RobotContainer which then asks the Subsystem to perform the specific action. This should probably be abstracted away from the Command in the sense that the Command is only talking to the RobotContainer and not having the RobotContainer pass the Subsystem through to the Command.

Reply:
When I looKed into how to interface with the subsystems, the examples I found passed the subsystems as a parameter. See https://github.com/wpilibsuite/allwpilib/tree/master/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot

Reply:
Yep, you beat me to it. The commands now have joint ownership of the subsystems (along with the RobotContainer) on which they operate and therefore do not have to go though the RobotContainer to access the Subsystem. I actually think that this is cleaner and retract my comment from earlier.